src/controllers/expenses.controller.js

```diff
@@ -0,0 +1,146 @@
+'use strict';
+
+const { ExpenseService } = require('../services/expenses.service');
+const { UserService } = require('../services/users.service');
+
+const getAll = (req, res) => {
+  let expenses = ExpenseService.getAll();
+  const {
+    userId,
+    categories,
+    from,
+    to,
+  } = req.query;
+
+  if (userId) {
+    expenses = expenses
+      .filter(expense => expense.userId === +userId);
```

Comment:
```suggestion
      .filter(expense => expense.userId === Number(userId));
```

The same in all other places